articles/services/performance-scalability.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Project to assist customers with common performance testing needs, including a correct planning, and test automation guidance.
+---
+
+# Geo High Availability
+
+An offering to assist customers with common performance testing needs, including a correct planning, and test automation guidance.
+
+## Estimated effort
+
+80 hours. 
+
+> Auth0 Professional Services are delivered in 40 hs. "blocks".
+
+## Typical work-streams
+
+* **Planning:** gather customer requirements for setting benchmarks.* **Performance Lab Design:** lab environment for testing, including network topology, artifacts required, load generation tools, automation scripts.
```

Comment:
@eugeniop You need to add an enter before the **Performance Lab Design** bullet. It displays like that:

* **Planning:** gather customer requirements for setting benchmarks.* **Performance Lab Design:** lab environment for testing, including network topology, artifacts required, load generation tools, automation scripts.